target/classes/banner.txt

```diff
@@ -0,0 +1,8 @@
+  _    _      _ _         __          __        _     _ _
```

Comment:
Add appengine/target/* to .gitignore file in the project root.

Reply:
Added appengine/target/* to .gitignore in the new commit.